### PR TITLE
[Navigator] Added transitionBackgroundColor property

### PR DESCRIPTION
**Pull request by PhilippKrone**
As discussed with @sahrens here: https://github.com/facebook/react-native/issues/2003

### CODE REVIEW

File: Libraries/CustomComponents/Navigator/Navigator.js [hunk 1]
```diff
@@ -1095,7 +1101,7 @@ var Navigator = React.createClass({
     return (
       <View style={[styles.container, this.props.style]}>
         <View
-          style={styles.transitioner}
+          style={[styles.transitioner,{backgroundColor: this.props.transitionBackgroundColor}]}
```
Comment (sahrens) [inline Libraries/CustomComponents/Navigator/Navigator.js]: space after comma, and make sure your lines aren't longer than 80 chars.  I'd format this like so:

  style={[
    styles.transitioner,
    {backgroundColor: this.props.transitionBackgroundColor}
  ]}

**Comment by PhilippKrone** [comment]
@sahrens done :)

**Comment by sahrens** [comment]
Can you address @ide's comment from #2003?  If you can just wrap your nested navigator in a view with the bg you need, I'd rather not add the complexity to the component here.

**Comment by sahrens** [comment]
@ide: please either close if we don't need this or accept and assign to me to import.

**Comment by PhilippKrone** [comment]
Yea, that seems to work.
@ide thanks!
@sahrens feel free to close that.

**Comment by sahrens** [comment]
Glad that worked :)

**Pull request closed without merging**